src/main/rx/cyrf6936_spektrum.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#define DSM_BIND_TIMEOUT_US 10000
+#define DSM_SYNC_TIMEOUT_US 20000
+#define DSM_RECV_LONG_TIMEOUT_US 18010
+#define DSM_RECV_MID_TIMEOUT_US 7000
+#define DSM_RECV_SHORT_TIMEOUT_US 4010
+#define DSM_RECV_TIMEOUT_OFFSET_US 1000
+
+#define DSM_MAX_MISSED_PACKETS 31
+
+#define DSM_MAX_RF_CHANNEL 0x4F
+#define DSM_MAX_CHANNEL_COUNT 12
+#define DSM_INITIAL_BIND_CHANNEL 0x0D
+
+#define DSM_TELEMETRY_FRAME_RPM 0x7E
+#define DSM_TELEMETRY_TIMEOUT_US 1500
+#define DSM_TELEMETRY_TIME_US 176000
+
+typedef enum {
+    DSM2_22     = 0x01,
+    DSM2_11     = 0x02,
+    DSM2_11_DX8 = 0x12,
+    DSMX_22     = 0xA2,
+    DSMX_11     = 0xB2,
+} dsm_protocol_e;
+#define IS_DSM2(x) (x == DSM2_22 || x == DSM2_11 || x == DSM2_11_DX8)
+#define IS_DSMX(x) (!IS_DSM2(x))
+
+#define CHECK_MFG_ID(protocol, packet, id) ((IS_DSM2(protocol) && packet[0] == (~id[2]&0xFF) && packet[1] == (~id[3]&0xFF)) || \
+        (IS_DSMX(protocol) && packet[0] == id[2] && packet[1] == id[3]))
+
+typedef struct spektrumConfig_s {
+    dsm_protocol_e protocol;
```

Comment:
Use an `uint8_t` here - enums are stored as 32 bit, which isn't needed in the parameter group.

Reply:
Done.